targets/targets.json

```diff
@@ -3630,12 +3635,14 @@
             }
         },
         "overrides": {
-            "lse_available": 0
+            "lse_available": 0,
+            "boot-stack-size": "0x400",
+            "tickless-from-us-ticker": true
```

Comment:
Why ?
Is it related to baremetal profile ?
Is it really needed as LPTICKER is not supported ?